shared/config/constants.json

```diff
@@ -0,0 +1,31 @@
+{
+  "enums": [{
+    "comment": "Used to communicate different types of levels",
+    "contents": {
+      "LevelKind": {
+        "peer_review": "peer_review",
+        "assessment": "assessment",
+        "puzzle": "puzzle",
+        "unplugged": "unplugged",
+        "level": "level"
+      }
+    }
+  },
+
+  {
+    "comment": "Different possibilities for level.status, used to communicate how user has performed on a given level",
```

Comment:
One downside to this schema is that it doesn't support multiline comments. Another is that there's currently an assumption that these are all enums, and if we wanted to share a naked constant, this doesn't currently provide a way to do so.